### ISSUE TITLE
Sometimes client Reconnect frequently

### ISSUE THREAD

**Issue by nkfeiyang** [SignalR/SignalR#4332]
I found my client sometimes received the connect state changed event frequently. But sometimes it is not found for several days. The client and server are both in same intranet network.

I am wondering, if there is some network problem could cause this?

Another issue is that sometimes after one client reconnected, it seems some data communication is not normal. Only after I restart the client, it becomes normal.

**Comment by clifton-collins3iii** [comment]
Possible duplicate or related to: https://github.com/SignalR/SignalR/issues/4321

**Comment by anurse** [comment]
Can you enable client-side tracing using [the guide in our documentation](https://docs.microsoft.com/en-us/aspnet/signalr/overview/testing-and-debugging/enabling-signalr-tracing#enabling-tracing-in-the-javascript-client) and post the log here? Please drag-and-drop a text file onto the comment box rather than pasting the log content here as it can be large. Be aware that the log may contain message content so if this is only reproducable in production we can provide contact information to send the logs privately.

**Comment by anurse** [comment]
Closing this as we haven't heard from you and generally close issues with no response after ~7 days. Please feel free to comment if you're able to get the information we're looking for and we can reopen the issue to investigate further!
